### PR TITLE
add form template explanation snow

### PR DESCRIPTION
**Pull request by naman-goenka**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screenshot.

### CODE REVIEW

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-service-now-tickets-in-cd-stages.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Make sure you have met the following requirements for searching form templates:
 - Integration for Harness Software Delivery Platform app version 1.0.2 or later.
 :::
 
-If there are many templates with the same name, the most current one is used to create tickets. Select **Apply Changes**.
+This option faciliates creation of tickets with values as defined in the linked form template of the associated table. This is achieved via scripted APIs defined in the ServiceNow integration app for Harness templates. If there are many templates with the same name, the most current one is used to create tickets. Select **Apply Changes**.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-service-now-tickets-in-cd-stages.md:97]:
```suggestion
This option creates tickets with values as defined in the linked form template for the associated table. This is achieved via scripted APIs defined in the ServiceNow integration app for Harness templates. If there are many templates with the same name, the most current one is used to create tickets. Select **Apply Changes**.
```

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md [hunk 2]
```diff
@@ -104,6 +104,8 @@ The timezone settings govern the display value of the settings not their actu
    All the fields corresponding to the Template are listed.
 3. Select **Apply Changes**.
 
+This option faciliates update of tickets with values as defined in the linked form template of the associated table. This is achieved via scripted APIs defined in the ServiceNow integration app for Harness templates.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md:107]:
```suggestion
This option creates tickets with values as defined in the linked form template for the associated table. This is achieved via scripted APIs defined in the ServiceNow integration app for Harness templates.
```

**Review by roshnisarangadharan — APPROVED** [review]
Approved with a minor update. Let me know when to merge.

File: docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md [hunk 3]
```diff
@@ -104,7 +104,7 @@ The timezone settings govern the display value of the settings not their actu
    All the fields corresponding to the Template are listed.
 3. Select **Apply Changes**.
 
-This option faciliates update of tickets with values as defined in the linked form template of the associated table. This is achieved via scripted APIs defined in the ServiceNow integration app for Harness templates.
+This option creates tickets with values as defined in the linked form template for the associated table. This is achieved via scripted APIs defined in the ServiceNow integration app for Harness templates.
```
Comment (naman-goenka) [inline docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/update-service-now-tickets-in-cd-stages.md:107]: @roshnisarangadharan this should be update

**Pull request merged**